job_offers_data_load/PostgreSQL_tables_declaration.py

```diff
@@ -0,0 +1,195 @@
+from sqlalchemy import Table, Column, Integer, String, ForeignKey
+from sqlalchemy.orm import relationship
+from sqlalchemy.ext.declarative import declarative_base
+import shortuuid
+
+Base = declarative_base()
+
+#  association Tables
+association_JobOffer_Location = Table('association_JobOffer_Location', Base.metadata,
```

Comment:
It is possible to add custom columns to association
https://docs.sqlalchemy.org/en/14/orm/basic_relationships.html#association-object